manifests/volume.pp

```diff
@@ -40,15 +40,16 @@
 # Copyright 2014 CoverMyMeds, unless otherwise noted
 #
 define gluster::volume (
+  Array $bricks,
+
   Boolean $force                              = false,
   Enum['tcp', 'rdma', 'tcp,rdma'] $transport  = 'tcp',
   Boolean $rebalance                          = true,
   Boolean $heal                               = true,
-  Array $bricks                               = undef,
   Boolean $remove_options                     = false,
   Optional[Array] $options                    = undef,
   Optional[Integer] $stripe                   = undef,
-  Optional[Integer] $replica                  = false,
+  Optional[Integer] $replica                  = undef,
```

Comment:
LGTM.  Lots of the code does `if $replica`, but that should work fine with a default of `undef`.